### PR TITLE
Fix for analytics query, flask-security config

### PR DESCRIPTION
**Pull request by tuz666**
Fixed uploaded episodes data statistics:

- Deleted redundant queries, fixed the logic and updated the swagger. 

Enabled flask-security's option to cache auth tokens. ([1](https://github.com/lahmacunradio/arcsi/pull/165/files#diff-5647f21b4c1e96dff15d3602cf623b1347909ad7084e8a04ed141408fa6f5fa8), [2](https://github.com/lahmacunradio/arcsi/pull/165/files#diff-a6ec850256c7656c9a92a833bd89cb3777971b7b2c77dba9f4b9b1e9754ebe32))

[Increased the number](https://github.com/lahmacunradio/arcsi/pull/165/files#diff-7829468e86c1cc5d5133195b5cb48e1ff6c75e3e9203777f6b2e379d9e4882b3) of newman tests run during Robustness CI tests

Arcsi's image [version uplift](https://github.com/lahmacunradio/arcsi/pull/165/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3)

### CODE REVIEW

File: arcsi/templates/data/uploaded_episodes.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{% extends 'base.html' %}
+
+{% block header %}
+<h2>{% block title %}Uploaded episodes{% endblock %}</h2>
+<h5>This query gives back the number of uploaded episodes in the selected timeframe.</h5>
+<h5>Both date are inclusive!</h5>
```
Comment (gammaw) [inline arcsi/templates/data/uploaded_episodes.html:6]: Both end dates are included!

File: arcsi/api/data.py [hunk 2]
```diff
@@ -8,43 +8,25 @@
...
-
-    start_date = end_date - timedelta(days=days)
-    episodes_count = Item.query.filter(
-        start_date < Item.play_date
-        ).filter(Item.play_date <= end_date).count()
-
-    ret = {"uploaded_episodes_between_%s_%s" % (start_date.date(), end_date.date()): episodes_count}
-    return make_response(jsonify(ret), 200, headers)
-
-@arcsi.route("/data/episodes_uploaded_weekly", methods=["POST"])
```
Comment (gammaw) [inline arcsi/api/data.py:29]: Makes sense to delete this, one query for this is enough.

**Review by gammaw — APPROVED** [review]
Good job! Thanks for the fix & happy hols! ;)
PS: you could really have created a new branch for this. It almost seems that we're obfuscating branch names here ;)

File: arcsi/api/data.py (hunk 2 above)
Comment (tuz666) [reply arcsi/api/data.py:29]: Yep, I already cleaned that on my local, I've just wanted to update the swagger docs as well and push it after that. Sorry for not putting it back to draft.

**Comment by tuz666** [comment]
Thanks, same to you!! 🤗
In the beginning I thought that both fix will be just few line changes.
The security still remained that long, but the analytics related turned out to be moore, I will update the PR description according to that.

File: arcsi/templates/data/uploaded_episodes.html (hunk 1 above)
Comment (tuz666) [reply arcsi/templates/data/uploaded_episodes.html:6]: Yepp, it's more clear that way, fixed it!

Comment (gammaw) [reply arcsi/templates/data/uploaded_episodes.html:6]: Hmm, it's still the old text (Both date are inclusive!). It's only grammer, so let's not block the PR and go with it.

**Review by gammaw — APPROVED** [review]
Happy New Year mister head of working group! ;)

**Comment by tuz666** [comment]
@gammaw I forgot to push my last commit which fixed this typo/grammar stuff...

**Review by gammaw — APPROVED** [review]

**Pull request merged**